src/gui/editors/PianoRoll.cpp

```diff
@@ -777,45 +666,34 @@ void PianoRoll::fitNoteLengths(bool fill)
 		{
 			length = (*ref)->pos() - note->pos();
 		}
-		if (fill ? note->length() < length : note->length() > length)
-		{
-			note->setLength(length);
-		}
+		if (fill ? note->length() < length : note->length() > length) { note->setLength(length); }
 	}
 
 	update();
 	getGUI()->songEditor()->update();
 	Engine::getSong()->setModified();
 }
 
-
 void PianoRoll::constrainNoteLengths(bool constrainMax)
 {
 	if (!hasValidMidiClip()) { return; }
 	m_midiClip->addJournalCheckPoint();
 
 	NoteVector notes = getSelectedNotes();
-	if (notes.empty())
-	{
-		notes = m_midiClip->notes();
-	}
+	if (notes.empty()) { notes = m_midiClip->notes(); }
 
-	TimePos bound = m_lenOfNewNotes;  // will be length of last note
+	TimePos bound = m_lenOfNewNotes; // will be length of last note
 	for (Note* note : notes)
 	{
-		if (constrainMax ? note->length() > bound : note->length() < bound)
-		{
-			note->setLength(bound);
-		}
+		if (constrainMax ? note->length() > bound : note->length() < bound) { note->setLength(bound); }
```

Comment:
It's in 1 line now and fits the 120 columns, so the tool does what we requested, but I'm not sure if it got more readable.